nautobot/core/templates/inc/media_failure.html

```diff
@@ -0,0 +1,111 @@
+                            into the static root path.
+                        </li>
+                        <li style="margin-bottom: 12px;">
+                            The HTTP service (e.g. NGINX) is configured to serve files from the <code>STATIC_ROOT</code> path.
+                            Refer to
+                            <a href="https://docs.nautobot.com/projects/core/en/v{{ settings.VERSION }}/installation/" target="_blank">
+                                the installation documentation
+                            </a>
+                            for further guidance.
+                            <ul>
+                                {% if request.user.is_staff or request.user.is_superuser %}
+                                    <li>
+                                        <code>
+                                            STATIC_ROOT: <strong>{{ settings.STATIC_ROOT }}</strong>
+                                        </code>
+                                    </li>
+                                {% endif %}
+                                <li>
+                                    <code>
+                                        STATIC_URL: <strong>{{ settings.STATIC_URL }}</strong>
+                                    </code>
+                                </li>
+                            </ul>
+                        </li>
+                        <li style="margin-bottom: 12px;">
+                            The file(s) listed above exists in the static root directory and is readable by the HTTP process.
+                        </li>
+                    </ul>
+                    <p>
+                        Click <a href="">here</a> to reload {{ settings.BRANDING_TITLE }}.
+                    </p>
+                </div>
+            `;
+
+            // If media failure alert already exists, just update its content and return early.
+            if (failureAlertRef.current) {
+                failureAlertRef.current.innerHTML = innerHTML;
+                return;
+            }
+
+            // If media failure alert does not exist yet, create one.
+            const alert = document.createElement('div');
+            alert.classList.add('alert', 'alert-danger', 'media-failure-banner');
+            alert.setAttribute('role', 'alert');
+            alert.innerHTML = innerHTML;
+
+            // Copy and hardcode Bootstrap `"alert alert-danger"` styles here to make the alert core styles independent of other CSS.
+            alert.style.setProperty('background-color', '#fce8e9');
+            alert.style.setProperty('border', '1px solid #f07162');
+            alert.style.setProperty('border-radius', '10px');
+            alert.style.setProperty('color', '#e01f1f');
+            alert.style.setProperty('margin', '20px');
+            alert.style.setProperty('margin-top', '0');
+            alert.style.setProperty('padding', '20px');
```

Comment:
Why not hard-code those inline to `innerHTML` above? We already have a number of other `style` declarations in there.